build.sbt

```diff
@@ -2,6 +2,12 @@ name := "root"
 
 Common.settings
 
+libraryDependencies ++= Seq(
```

Comment:
Should dependancies be specified in the top level build file? Or in the projects themselves